R/theme-defaults.r

```diff
@@ -618,7 +618,8 @@ theme_test <- function(base_size = 11, base_family = "",
                            margin = margin(0.8 * half_line, 0.8 * half_line, 0.8 * half_line, 0.8 * half_line)
                          ),
     strip.text.x =       NULL,
-    strip.text.y =       element_text(angle = -90),
+    strip.text.y = element_text(angle = -90),
+    strip.text.y.left = element_text(angle = 90),
```

Comment:
Thanks, almost perfect! Sorry for nitpicking, could you align these `element_text()` to the other lines?

Reply:
I understand the need for consistency; I just updated it.